lib/features/feed/cubit/feed_cubit.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:bloc/bloc.dart';
+import 'package:fluttercon/common/data/models/feed.dart';
+import 'package:fluttercon/common/data/models/failure.dart';
+import 'package:fluttercon/common/repository/api_repository.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'feed_state.dart';
+part 'feed_cubit.freezed.dart';
+
+class FetchFeedCubit extends Cubit<FetchFeedState> {
+  FetchFeedCubit({
+    required ApiRepository apiRepository,
+  }) : super(const FetchFeedState.initial()) {
+    _apiRepository = apiRepository;
+  }
+
+  late ApiRepository _apiRepository;
+
+  Future<void> fetchFeeds() async {
+    emit(const FetchFeedState.loading());
+    try {
+      final feeds = await _apiRepository.fetchFeeds(event: '');
```

Comment:
The slug for the feed of this event as per the documentation at this URL should be `droidconke-2022-281`

https://documenter.getpostman.com/view/3385291/SzS4TTXb#a443ec01-1cf8-41d9-bfd5-56fb6c228de4

Please make this slight addition.